know/rdf/parser/parser.py

```diff
@@ -0,0 +1,42 @@
+import re
+from .wikidata_parser import WikidataParser
+from .viaf_parser import ViafParser
+class Parser:
+    """parses all sources"""
```

Comment:
```suggestion
    """ parses all sources """
```
Add spaces around this, and make it more descriptive